packages/pocket-ic/src/lib.rs

```diff
@@ -867,6 +868,27 @@ impl PocketIc {
                 .await
         })
     }
+
+    /// Get the pending canister HTTP outcalls.
+    #[instrument(ret, skip(self), fields(instance_id=self.pocket_ic.instance_id))]
+    pub fn get_canister_http(&self) -> Vec<CanisterHttpRequest> {
```

Comment:
Do I understand correctly that when the user's canister makes outcalls, the test code has to fetch these requests "manually" and supply a response to these pending calls? Otherwise the outcalls never "return"?

Reply:
Yes, in the live mode, this happens automatically, otherwise the test driver needs to do it, similarly to the `test_canister_http` library test.

Reply:
Also remarked this in a doc comment next to `get_canister_http`.